deploy/marlowe-playground.yaml

```diff
@@ -0,0 +1,163 @@
+        value: http://localhost:8080
+      - name: JWT_SIGNATURE
+        valueFrom:
+          secretKeyRef:
+            key: JWT_SIGNATURE
+            name: jwt-signature
+      - name: GITHUB_CLIENT_ID
+        valueFrom:
+          secretKeyRef:
+            key: clientID
+            name: gh-oauth
+      - name: GITHUB_CLIENT_SECRET
+        valueFrom:
+          secretKeyRef:
+            key: clientSecret
+            name: gh-oauth
+      - name: GITHUB_CALLBACK_PATH
+        valueFrom:
+          secretKeyRef:
+            key: callbackPath
+            name: gh-oauth
+      image: ghcr.io/input-output-hk/marlowe-playground-server:latest
+      imagePullPolicy: Always
+      memory: 1024Mi
+      ports:
+      - expose: true
+        port: 8080
+        protocol: TCP
+    traits:
+    - properties:
+        replicas: 1
+      type: scaler
+    - properties:
+        domains:
+        - marlowe-playground.scdev.aws.iohkdev.io
+        rules:
+        - path:
+            type: PathPrefix
+            value: /api
+          port: 8080
+      type: https-route
+    type: webservice
+  - name: marlowe-playground-client
+    properties:
+      cpu: "0.5"
+      image: ghcr.io/input-output-hk/marlowe-playground-client:latest
+      imagePullPolicy: Always
+      memory: 1024Mi
+      ports:
+      - expose: true
+        port: 8080
+        protocol: TCP
+    traits:
+    - properties:
+        replicas: 1
+      type: scaler
+    - properties:
+        domains:
+        - marlowe-playground.scdev.aws.iohkdev.io
+        rules:
+        - port: 8080
+      type: https-route
+    type: webservice
+  policies:
+  - name: local-marlowe-staging
+    properties:
+      clusters:
+      - local
+      namespace: marlowe-staging
+    type: topology
+
+  - name: local-marlowe-production
+    properties:
+      clusters:
+      - local
+      namespace: marlowe-production
+    type: topology
+
+  - name: override-marlowe-production
+    type: override
+    properties:
+      components:
+      - name: marlowe-playground
+        properties:
+          env:
+          - name: FRONTEND_URL
+            value: https://play.marlowe.iohk.io
+          - name: WEBGHC_URL
+            value: http://localhost:8080
+          - name: JWT_SIGNATURE
+            valueFrom:
+              secretKeyRef:
+                key: JWT_SIGNATURE
+                name: jwt-signature
+          - name: GITHUB_CLIENT_ID
+            valueFrom:
+              secretKeyRef:
+                key: clientID
+                name: gh-oauth-production
+          - name: GITHUB_CLIENT_SECRET
+            valueFrom:
+              secretKeyRef:
+                key: clientSecret
+                name: gh-oauth-production
+          - name: GITHUB_CALLBACK_PATH
+            valueFrom:
+              secretKeyRef:
+                key: callbackPath
+                name: gh-oauth-production
+          image: ghcr.io/input-output-hk/marlowe-playground-server:2023-05-30-e2d5fd6
```

Comment:
Is this meant to be hardcoded?

Reply:
Ah, it is because it is production?

Reply:
Yes I was told there would be a production image, but thats not there yet so I just hardcoded it at the version Jamie deployed to production. Once the production image is available someone should change it.